libmscore/edit.cpp

```diff
@@ -4302,6 +4302,9 @@ void Score::undoAddElement(Element* element)
                         ne->setParent(m);
                         undo(new AddElement(ne));
                         }
+                  else if (et == ElementType::MEASURE_NUMBER) {
+                        toMeasure(element->parent())->undoChangeProperty(Pid::MEASURE_NUMBER_MODE, (int) MeasureNumberMode::SHOW);
```

Comment:
Can ypou please avoid those old style C casts? At least use the C++ function style casts `int(MeasureNumberMode::SHOW)`